source/initial_temperature/interface.cc

```diff
@@ -204,7 +204,7 @@ namespace aspect
                           std::get<dim>(registered_plugins).get_description_string());
 
         prm.declare_entry("List of model operators", "add",
-                          Patterns::MultipleSelection("add|subtract|minimum|maximum"),
+                          Patterns::MultipleSelection("add|subtract|minimum|maximum|replace if valid"),
```

Comment:
We should document "somewhere" what these operators actually are. Is there a good place for this?